OMCompiler/Compiler/FrontEnd/MetaModelicaBuiltin.mo

```diff
@@ -387,6 +387,14 @@ algorithm
   annotation(__OpenModelica_EarlyInline = true, __OpenModelica_BuiltinPtr = true);
 end realAbs;
 
+function realAlmostEq
+  input Real a;
+  input Real b;
+  input Real absTol = 1e-6;
+  output Boolean c = realGt(absTol, realAbs(a-b));
+  annotation(__OpenModelica_EarlyInline = true, __OpenModelica_BuiltinPtr = true);
```

Comment:
```mo
function realAlmostEq
  input Real r1;
  input Real r2;
  input Real t = 1e-6;
  output Boolean r;
algorithm
  r := t > realAbs(r1 - r2);
  annotation(__OpenModelica_EarlyInline = true, __OpenModelica_BuiltinPtr = true);
end realAlmostEq;

```
Seems to work for me. There is definitely an in-lining issue with the original one.